roles/setup_gitea/templates/route_gitea.j2

```diff
@@ -0,0 +1,13 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: gitea
+  namespace: "{{ sg_namespace }}"
+spec:
+  port:
+    targetPort: http
+  to:
+    kind: Service
+    name: gitea
+    weight: 100
+  wildcardPolicy: None
```

Comment:
Missed line break at the end of the file

Reply:
Done